src/services/local/index.js

```diff
@@ -105,7 +103,7 @@ export class Service {
 }
 
 export default function(options){
-  options = Object.assign({}, defaults, options);
+  options = Object.assign({ passReqToCallback: true }, defaults, options);
```

Comment:
If you look to `checkCredentials`, the first parameter is now the request.

Passport mentions it in http://passportjs.org/docs/configure#association-in-verify-callback

> One downside to the approach described above is that it requires two instances of the same strategy and supporting routes.
> 
> To avoid this, set the strategy's passReqToCallback option to true. With this option enabled, req will be passed as the first argument to the verify callback.